server/app/util/logger.js

```diff
@@ -0,0 +1,14 @@
+import { Logger, transports } from 'winston';
+import config from '../config';
+
+const logger = new Logger({
+  transports: [
+    new (transports.Console)({
+      level: config.logLevel,
+      colorize: true
+    })
+    /* new (transports.File)()({ filename: 'somefile.log' }) */
```

Comment:
This is important for logging purposes. Down the road, we can tie this into a logging service, but for now it just allows us to configure what environment log statements can show up, and at what level (debug, info, warn, error, etc.). It's a little more power than `console.log` and a much better practice.